src/bench/inputfetcher.cpp

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2024-present The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <bench/bench.h>
+#include <bench/data/block413567.raw.h>
+#include <coins.h>
+#include <common/system.h>
+#include <inputfetcher.h>
+#include <primitives/block.h>
+#include <serialize.h>
+#include <streams.h>
+#include <util/time.h>
+
+static constexpr auto QUEUE_BATCH_SIZE{128};
```

Comment:
In 2349ac7d6071746a80223358bce0d5e556b277d7 "bench: add inputfetcher bench"
How did you select this batch size?

Reply:
This is the hardcoded batch size used in CheckQueue. Not sure why that was selected, but I deferred to previous choices.